import java.util.InputMismatchException;
import java.util.Scanner;

public class TryAndCatchExample
{
	public static void main(String[] args) {
		Scanner s=new Scanner(System.in);
		try
		{
			System.out.println("Enter 2 no's:");
			int a=s.nextInt();
			int b=s.nextInt();
			int c=a/b;//----Create object of ArithmeticException--throw
			System.out.println("Division is :"+c);
		}
		System.out.println("Hello");
		catch (ArithmeticException e)
		{
			System.out.println(e);//toString()
			System.out.println("Can't devide ob zero");
		}
		catch (InputMismatchException e) {
			System.out.println(e);
		}
		System.out.println("Rest of the code.....");
	}

}
